my-app/src/app/blog/about-us/about-us.component.css

```diff
@@ -0,0 +1,9 @@
+div {
```

Comment:
Remember to use :host, instead of a styled div on the outside of everything in the component.